decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
```

Comment:
- Let's also remove the super as this is currently not needed.

Reply:
OK, it's done! thanks!

Reply:
But now I recieved the following offenses from rubocop:

decorator.rb:4:3: W: Lint/MissingSuper: Call super to initialize state of the parent class.
  def initialize(nameable) ...
  ^^^^^^^^^^^^^^^^^^^^^^^^
person.rb:4:3: W: Lint/MissingSuper: Call super to initialize state of the parent class.
  def initialize(age, name = 'Unknown', parent_permission: true) ...
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^